### DISCUSSION TITLE
能否在登录页面自定义页脚

### DISCUSSION THREAD

**Discussion by ka0ri39** [hslr-s/sun-panel#262] [General]
如题，我想添加备案号但是我又不想开放游客登录

**Comment by Coco-wxf** [comment]
1、你先将网站挂上，然后检查-搜索“Powered”，已我自己为例，找到两个.js包含“Powered”，然后将这两个js文件记一下，index-01454222.js，index-a6e98062.js,如图1所示。
![1](https://github.com/user-attachments/assets/eed1e11b-e229-4ac4-baa2-a7e6997fbc5c)

2、去app目录，将内容替换掉，“ Powered by ” 替换成 ”某ICP备 “ & 将 网址替换为”https://beian.miit.gov.cn/“ & 将 部分 "sun-panel" 替换成 ”**********号“ （因为下面有很多注释，没必要全局替换，替换几个就行）,然后重启一下，就行了。如图二、三所示。
![2](https://github.com/user-attachments/assets/6ecda803-53f0-4e30-89d9-369193cce0e5)
![3](https://github.com/user-attachments/assets/44fb8b59-35a9-4e5c-89ef-a747346a9fd6)
